src/outputs/frozen_group.py

```diff
@@ -0,0 +1,129 @@
+"""The main file for the frozen group output."""
+
+import logging
+import pandas as pd
+from datetime import datetime
+from typing import Callable, Dict, Any
+
+from src.outputs.outputs_helpers import aggregate_output, create_output_df
+import src.outputs.map_output_cols as map_o
+from src.staging.validation import load_schema
```

Comment:
Import should be re-organised to comply with PEP-8.

The correct order for imports should be:
1. Default python packages (part of the standard library)
2. Third party python packages
3. Local imports

In this case, this would look like the following:
```python
import logging
from typing import Callable, Dict, Any

import pandas as pd
from datetime import datetime

from src.outputs.outputs_helpers import aggregate_output, create_output_df
import src.outputs.map_output_cols as map_o
from src.staging.validation import load_schema
```

Reply:
Good point, corrected